metricbeat/module/mongodb/dbstats/_meta/docs.asciidoc

```diff
@@ -1 +1,44 @@
 This is the `dbstats` metricset of the MongoDB module.
+
+It requires the following privileges:
+
+- https://docs.mongodb.com/manual/reference/privilege-actions/#listDatabases[`listDatabases` action] on https://docs.mongodb.com/manual/reference/resource-document/#cluster-resource[`cluster` resource]
+- for each of the databases, also need https://docs.mongodb.com/manual/reference/privilege-actions/#dbStats[`dbStats` action] on the https://docs.mongodb.com/manual/reference/resource-document/#database-and-or-collection-resource[`database` resource]
+
+In Mongo shell, you can use the following command to create the role
+
+....
+db.createRole(
+   {
+     role: "dbstatsRole",
+     privileges: [
+       { resource: { cluster: true }, actions: ["listDatabases"] },
+       // for each of the databases
+       { resource: { db: "db_1"}, actions: ["dbStats"]},
```

Comment:
🤔 this one is not working for me giving: `E  QUERY    [js] uncaught exception: Error: resource must set both db and collection or neither, but not exactly one. :`

Had to change it to:
```
db.createRole(
   {
     role: "dbstatsRole",
     privileges: [
       { resource: { cluster: true }, actions: ["listDatabases"] },
       { resource: { db: "testdb", collection: ""}, actions: ["dbStats"]}
     ],
     roles: []
   }
)
```